WordPress/Classes/Models/AbstractPost.h

```diff
@@ -64,6 +64,8 @@ typedef NS_ENUM(NSUInteger, AbstractPostRemoteStatus) {
 @property (nonatomic, strong, nullable) NSString *confirmedChangesHash;
 @property (nonatomic, strong, nullable) NSDate *confirmedChangesTimestamp;
 
+@property (nonatomic, strong, nullable) NSString *voiceContent;
```

Comment:
I didn't want to change the editor's API for this change. It's probably no an ideal API design, but should suffice for this feature.